manifests/install/charts/as-a-second-scheduler/README.md

```diff
@@ -1 +1,45 @@
-# Chart to run scheduler plugin as a second scheduler in cluster.
+# Scheduler-plugins as a second scheduler in cluster
+
+## Installation
+
+Quick start instructions for the setup and configuration of as-a-second-scheduler using Helm.
+
+### Prerequisites
+
+- [Helm](https://helm.sh/docs/intro/quickstart/#install-helm)
+
+### Installing the chart
+
+
+#### Install chart using Helm v3.0+
+
+```bash
+$ git clone git@github.com:kubernetes-sigs/scheduler-plugins.git
+$ cd scheduler-plugins/manifests/install/charts
+$ helm install scheduler-plugins as-a-second-scheduler/
+```
+
```

Comment:
````suggestion
Verify that scheduler and plugin-controller pod are running properly.

```bash
$ kubectl get deploy -n scheduler-plugins
NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
scheduler-plugins-controller   1/1     1            1           7s
scheduler-plugins-scheduler    1/1     1            1           7s
```
````